src/collection/layout.js

```diff
@@ -26,6 +26,20 @@ var elesfn = ({
           node.silentPosition( { x: 0, y: 0 } );
         }
 
+        if ( options.spacingFactor && options.spacingFactor !== 1){
+          var spacing = Math.abs(options.spacingFactor);
+          var nbb = nodes.boundingBox();
+          var center = {
+            x: nbb.x1 + nbb.w / 2,
+            y: nbb.y1 + nbb.h / 2
+          };
+          var spacingVector = { // scale the spacing from center of bounding box (not necessarily 0,0)
+            x: (newPos.x - center.x) * spacing,
+            y: (newPos.y - center.y) * spacing
+          };
+          newPos = util.extend( {}, newPos, { x: center.x + spacingVector.x, y: center.y + spacingVector.y } );
```

Comment:
Thanks. Wasn't sure if the newPos object had other properties I would wipe by reassigning it.